examples/pylab_examples/print_stdout.py

```diff
@@ -1,13 +1,9 @@
-#!/usr/bin/env python
-# -*- noplot -*-
-# print png to standard out
```

Comment:
What does the 2nd line tell us? Btw, I'm adding them back anyways, but just curious.